docs/contents/components/forms/year-picker/index.ja.mdx

```diff
@@ -28,59 +28,95 @@ import { YearPicker } from "@yamada-ui/calendar"
 
 ```tsx
 <VStack>
```

Comment:
@kaehehehe 
It seems that using `<For />` has actually reduced readability. Why don't we try replacing `<For />` with something else?

Reply:
@koralle 
I think it's better to use the `For` component. Why do you think the readability is lower?